ts/Iterable.ts

```diff
@@ -30,6 +30,11 @@ enum IterableActionSource {
   inApp = 2
 }
 
+enum AuthResponseCallback {
+  SUCCESS = 'success',
```

Comment:
Do you need a string value for the enum defined?

Reply:
Makes sense. Removed the string value and tested and it still works. Thanks :)